docs/multi-cluster-mesh/multi-cluster-mesh.md

```diff
@@ -0,0 +1,81 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
+Setup three clusters (one active – two passive) in a single mesh multi network (two in the same network, one in a different network) setup with Istio operator.
```

Comment:
I'd create separate guides for primary-primary and primary-remote setups. It'll be easier to understand for the users.